gradyrobbins/fifty
# Local api no heroku

# Description
fixes issue where userId was not parsed into an integer

## Type of change

Please delete options that are not relevant.



# Testing Instructions for Change Made
open DevTools; navigate to <AddaQuarterForm /> component; verify that the "state" of this component reflects the current user's ID expressed as an integer.  

# Checklist:
